### PR TITLE
Start oleaut32 with a few functions

### PR DESCRIPTION
**Pull request by bvinc83**

### CODE REVIEW

**Comment by retep998** [comment]
Why are a bunch of the functions commented out even though they have full signatures?

**Comment by CruzBishop** [comment]
Perhaps not 100% certain?

**Comment by bvinc83** [comment]
I did all of the work for their signature, but not the work to implement the crazy complicated types like VARIANT.  The really important ones to have are the first ones like SysAllocString.

File: lib/oleaut32-sys/src/lib.rs [hunk 1]
```diff
@@ -1,9 +1,382 @@
 // Copyright © 2015, Peter Atashian
 // Licensed under the MIT License <LICENSE.md>
-//! FFI bindings to oleaut32.
-#![no_std]
-#![experimental]
+//! FFI bindings to vssapi.
```
Comment (retep998) [inline lib/oleaut32-sys/src/lib.rs]: Are you sure you intended to change this comment?

**Comment by bvinc83** [comment]
Good catch on the comment.  Fixed and rebased to master.

**Pull request merged**